libmei/mei/mei-basic_compiled.odd

```diff
@@ -6041,7 +6041,7 @@
     </constraintSpec><attList><attDef ident="meiversion" usage="req"><desc xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">Specifies a generic MEI version label.</desc><valList type="closed" mode="change"><valItem xmlns:xi="http://www.w3.org/2001/XInclude" ident="5.1">
             <desc xml:lang="en">Version of MEI</desc>
           </valItem><valItem xmlns:xi="http://www.w3.org/2001/XInclude" ident="5.1+basic">
-              <desc>Version of MEI basic customization</desc>
+              <desc>Version of MEI Basic customization</desc>
```

Comment:
Should be fixed in the source file, not the odd

Reply:
True, I thought it has been already.

Reply:
reverted.